package.json

```diff
@@ -40,6 +40,7 @@
     "react-native-animatable": "0.6.1",
     "react-native-button": "1.6.0",
     "react-native-calendar": "0.6.4",
+    "react-native-cli": "1.0.0",
```

Comment:
This is in response to the issues that I was having building when I pulled from master. This is the windows CLI implementation that you need to build with the command `react-native run-android`. (I think)